### PR TITLE
Enable serilog COUTH-5200

### PR DESCRIPTION
**Pull request by Sacrelicious**

### CODE REVIEW

File: Raven/appsettings.json [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+    "MinimumLevel": {
+      "Default": "Information",
+      "Override": {
+        "Microsoft": "Warning",
+        "System": "Warning"
+      }
+    },
+    "WriteTo": [
+      {
+        "Name": "RollingFile",
```
Comment (dtoc) [inline Raven/appsettings.json]: Out of curiosity, why this name?

File: Raven/appsettings.json [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+          "pathFormat": "/logs/raven-{Date}.log",
+          "formatter": "Serilog.Formatting.Compact.RenderedCompactJsonFormatter, Serilog.Formatting.Compact"
+        }
+      }
+    ]
   },
   "AllowedHosts": "*",
-  "CaracaraApiUrl": "https://local-caracara.choice.plumbing/"
+  "CaracaraApiUrl": "https://local-caracara.choice.plumbing/",
+  "Version": "(shrug)"
```
Comment (dtoc) [inline Raven/appsettings.json]: Do we need to provide a version if we don't care what version it is?

File: Raven/Startup.cs [hunk 3]
```diff
@@ -59,6 +63,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseHttpsRedirection();
             app.UseMvc();
+            //app.UseRaygun();
```
Comment (dtoc) [inline Raven/Startup.cs]: wat

File: Raven/appsettings.json (hunk 1 above)
Comment (Sacrelicious) [reply Raven/appsettings.json]: This tells serilog what appender to use, in this case, the "RollingFileAppender" which writes to a file that is then picked up by splunk

File: Raven/appsettings.json (hunk 2 above)
Comment (Sacrelicious) [reply Raven/appsettings.json]: This will be replaced with the version on deploy

File: Raven/Startup.cs (hunk 3 above)
Comment (Sacrelicious) [reply Raven/Startup.cs]: (curses) I thought I deleted all of these

**Pull request merged**